### ISSUE TITLE
Bug: WYSIWYG NPE if it finds an invalid data-elements

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#23296]
1. Go to the demo site.  
2. Create a new Rich Text content object, switch to text and paste this code:

```
<p><img src="https://www.dotcms.com/dA/90ddf569e5/Multi-Tenant-CMS-graphic.png?language_id=1" alt="Multi-Tenant-CMS-graphic.png" data-field-name="asset" data-inode="9af0b34d-51ab-45e1-9faa-03b7200003a6" data-identifier="90ddf569e5b0b9f8866049c0d4b8d6ba" data-saveas="Multi-Tenant-CMS-graphic.png" width="1201" height="596" /></p>
```
3. Save the content.
4. Try to reopen and edit it.  You will get an NPE:
```
Caused by: java.lang.NullPointerException
	at java.util.Objects.requireNonNull(Objects.java:221) ~[?:?]
	at java.util.Optional.<init>(Optional.java:107) ~[?:?]
	at java.util.Optional.of(Optional.java:120) ~[?:?]
	at com.dotmarketing.portlets.browser.BrowserUtil.resolveWithCurrentValue(BrowserUtil.java:171) ~[classes/:?]
	at com.dotmarketing.portlets.browser.BrowserUtil.lambda$0(BrowserUtil.java:49) ~[classes/:?]
	at com.dotmarketing.portlets.browser.BrowserUtil.getDefaultPathFolder(BrowserUtil.java:316) ~[classes/:?]
	at com.dotmarketing.portlets.browser.BrowserUtil.getDefaultPathFolderPathIds(BrowserUtil.java:350) ~[classes/:?]
```

**Comment by wezell** [comment]
PR #23298

**Comment by bryanboza** [comment]
Fixed, tested on release-22.11 // Docker // FF. And I'm unable to reproduce after this change.
